clients/osv.go

```diff
@@ -38,7 +39,10 @@ func (v osvClient) ListUnfixedVulnerabilities(
 ) (_ VulnerabilitiesResponse, err error) {
 	defer func() {
 		if r := recover(); r != nil {
-			err = sce.CreateInternal(sce.ErrScorecardInternal, fmt.Sprintf("osv-scanner panic: %v", r))
+			err = sce.CreateInternal(
+				sce.ErrScorecardInternal,
+				fmt.Sprintf("osv-scanner panic: %v\n%s", r, string(debug.Stack())),
+			)
```

Comment:
This error ends up in the published results, API, etc, as you noted in #4171. I don't think this level of debugging info belongs there. 

We can still return the internal error saying osv-scanner had a panic, and put the stack info in stderr instead?

Reply:
If the issue occurs in GitHub Actions, you could look for it in the GitHub action output. In our weekly scan infrastructure, stderr goes to our logging and metrics where I can search for it and grab the stack trace with the osv-scanner team.